apps/web/src/components/Publication/SinglePublication.tsx

```diff
@@ -76,7 +77,11 @@ const SinglePublication: FC<Props> = ({
           <HiddenPublication type={publication.__typename} />
         ) : (
           <>
-            <PublicationBody publication={rootPublication as LensterPublication} />
+            {publication?.metadata?.encryptionParams ? (
+              <DecryptedPublicationBody publication={rootPublication as LensterPublication} />
+            ) : (
+              <PublicationBody publication={rootPublication as LensterPublication} />
+            )}
```

Comment:
Why not infer them both at one place sir? 😅

Reply:
Fixed here 🙇🏼 

https://github.com/lensterxyz/lenster/pull/1393/commits/5062908978f7b2a91953523a8544067b3e8d02f6